docs/index.md

```diff
@@ -1,215 +1,54 @@
-</script>
-
-## Installation
-
-=== "pip"
-
-    ```sh
-    pip install ibis-framework
-    ```
-
-{% for mgr in ["conda", "mamba"] %}
-=== "{{ mgr }}"
-
-    ```sh
-    {{ mgr }} install -c conda-forge ibis-framework
-    ```
-
-{% endfor %}
-
-Try it out!
+Data analytics at scale: the expressiveness of Python, with the power and performance of modern SQL database engines.
 
 ```python
-python -c 'import ibis; print(ibis.__version__)'
-```
-
-## Features
-
-### SQL Coverage
-
-#### Anything you can write in a `SELECT` statement you can write in Ibis.
-
-=== "Group By"
-
-    ##### SQL
-
-    ```sql
-    SELECT f, sum(a + b) AS d
-    FROM t
-    GROUP BY f
-    ```
-
-    ##### Ibis
-
-    ```python
-    t.group_by("f").aggregate(d=(t.a + t.b).sum())
-    ```
-
-=== "Join"
-
-    ##### SQL
-
-    ```sql
-    SELECT exp(t.a) AS d
-    FROM t
-    LEFT SEMI JOIN s
-      ON t.x = t.y
-    ```
-
-    ##### Ibis
-
-    ```python
-    t.semi_join(s, t.x == t.y).select([lambda t: t.a.exp().name("d")])
-    ```
-
-=== "Window Functions"
-
-    ##### SQL
-
-    ```sql
-    SELECT *, avg(x) OVER (PARTITION BY y) as z
-    FROM t
-    ```
-
-    ##### Ibis
-
-    ```python
-    t.group_by("y").mutate(z=t.x.avg())
-    ```
-
-!!! tip "Coming from SQL?"
-
-    Check out [Ibis for SQL Programmers](ibis-for-sql-programmers)
-
-### Abstract Over SQL Dialects
-
-#### No more rewrites when scaling up or down.
+import ibis
 
-=== "SQLite"
+con = ibis.connect('movielens.sqlite')
 
-    ```python
-    con = ibis.sqlite.connect("my_sqlite.db")
-    ```
+movies = con.tables.movies
+rating_by_year = movies.group_by('year').avg_rating.mean()
+q = rating_by_year.order_by(query.year.desc())
 
-=== "PostgreSQL"
-
-    ```python
-    con = ibis.postgres.connect(user="me", host="my_computer", port=9090)
-    ```
-
-=== "BigQuery"
-
-    ```python
-    con = ibis.bigquery.connect(project_id="my_project_id", dataset_id="my_dataset_id")
-    ```
-
-```python
-t = con.table("t")
-t.group_by("y").mutate(z=t.x.avg())
+print(con.compile(q))
 ```
 
-### Ecosystem
-
-#### Ibis builds on top of and works with existing Python tools.
-
-```python
-expr = t.semi_join(s, t.x == t.y).select([lambda t: t.a.exp().name("d")]).head(2)
-df = expr.execute()  # a pandas DataFrame!
+```sql
+SELECT year, avg(avg_rating) FROM movies
+    GROUP BY t1.year
+    ORDER BY year DESC
 ```
 
-## Example
-
-Let's compute the number of citizens per squared kilometer in Asia:
-
-```python
->>> import ibis
->>> db = ibis.sqlite.connect("geography.db")
->>> countries = db.table("countries")
->>> asian_countries = countries.filter(countries.continent == "AS")
->>> density_in_asia = asian_countries.population.sum() / asian_countries.area_km2.sum()
->>> density_in_asia.execute()
-130.7019141926602
+```
+print(q.head().execute())
+
+     year  mean(avg_rating)
+0    2021          2.586362
+1    2020          2.719994
+2    2019          2.932275
+3    2018          3.005046
+4    2017          3.071669
```

Comment:
Can we make this all look like a repl session? As you have it here it feels like it lacks continuity.

```python
>>> import ibis
>>> con = ibis.connect('movielens.sqlite')
>>> movies = con.tables.movies
>>> rating_by_year = movies.group_by('year').avg_rating.mean()
>>> q = rating_by_year.order_by(query.year.desc())
>>> print(con.compile(q))
SELECT
  year,
  AVG(avg_rating)
FROM movies
GROUP BY t1.year
ORDER BY year DESC
>>> print(q.head().execute())
     year  mean(avg_rating)
0    2021          2.586362
1    2020          2.719994
2    2019          2.932275
3    2018          3.005046
4    2017          3.071669
```

Reply:
Could also remove the `print` from the `execute` call since DataFrames will repr like that.

Reply:
I was trying to avoid a REPL session, I was hoping to make it look cut-n-pasteable.  I tried something with `title=""`, let me know if that looks any better.